### PR TITLE
Many build infra fixes.

### PR DESCRIPTION
**Pull request by oliverchang**
- Remove base image project argument from deploy.sh. Deploy all
  functions to the main oss-fuzz image.
- Reduce max instances of functions to 1 to avoid rate limiting issues.
- Fix missing ndb context initialization in request_build.py
- Fix incorrect tags in BuildHistory entities. "-" was doubled.
- Fix base build deployment with incorrect schedule variable.
- Add scripts for requesting builds locally.

### CODE REVIEW

**Comment by oliverchang** [comment]
TBR to get ball rolling. @kabeer27 FYI.

File: infra/build/functions/base_images.py [hunk 1]
```diff
@@ -22,13 +22,15 @@
...
+
 
 # pylint: disable=no-member
 def base_builder(event, context):
   """Cloud function to build base images."""
   del event, context
-  credentials, project_id = google.auth.default()
-  tag_prefix = f'gcr.io/{project_id}/'
+  credentials, _ = google.auth.default()
+  tag_prefix = f'gcr.io/{BASE_PROJECT}/'
```
Comment (inferno-chromium) [inline infra/build/functions/base_images.py:33]: Did you forget a .format here ?

Comment (oliverchang) [reply infra/build/functions/base_images.py:33]: This is a Python 3 language feature :) https://realpython.com/python-f-strings/

File: infra/build/functions/deploy.sh [hunk 2]
```diff
@@ -83,19 +83,19 @@ function deploy_cloud_function {
 	--trigger-topic $topic \
 	--runtime python37 \
 	--project $project \
-	--timeout 540
+	--timeout 540 \
+  --max-instances 1
```
Comment (Dor1s) [inline infra/build/functions/deploy.sh:87]: nit: looks like indentation above was using tabs

**Pull request merged**